src/sql/base/browser/ui/selectBox/media/selectBox.css

```diff
@@ -0,0 +1,10 @@
+.labelOnTopContainer {
```

Comment:
Minor: should we be scoping this at all?

Reply:
These are global CSS values, hence why we often scope if possible. Not vital given they seem unique.